### PR TITLE
data cutter logging

### PR DESCRIPTION
**Pull request by AdamChit**
**Related issues**
Refer to issue(s) addressed in this pull request from [Issues](https://github.com/salesforce/TransmogrifAI/issues) page.

**Describe the proposed solution**
A clear and concise description of what the changes are.

**Describe alternatives you've considered**

**Additional context**
Add any other context about the changes here.

### CODE REVIEW

File: core/src/main/scala/com/salesforce/op/stages/impl/tuning/DataCutter.scala [hunk 1]
```diff
@@ -125,9 +125,9 @@ class DataCutter(uid: String = UID[DataCutter]) extends Splitter(uid = uid) with
     val dataPrep = data
       .filter(r => labelSet.contains(r.getDouble(labelColIdx)))
       .withColumn(labelColName, data(labelColName).as(labelColName, metadataNA.toMetadata))
-
+    log.info(s"number of rows after only using top 100 labels ${dataPrep.count()}")
     // calculate the down sample fraction
-    val dataSetSize = data.count()
+    val dataSetSize = dataPrep.count()
```
Comment (tovbinm) [inline core/src/main/scala/com/salesforce/op/stages/impl/tuning/DataCutter.scala]: so this was a bug then?

File: core/src/main/scala/com/salesforce/op/stages/impl/tuning/DataCutter.scala [hunk 2]
```diff
@@ -125,9 +125,9 @@ class DataCutter(uid: String = UID[DataCutter]) extends Splitter(uid = uid) with
     val dataPrep = data
       .filter(r => labelSet.contains(r.getDouble(labelColIdx)))
       .withColumn(labelColName, data(labelColName).as(labelColName, metadataNA.toMetadata))
-
+    log.info(s"number of rows after only using top 100 labels ${dataPrep.count()}")
```
Comment (tovbinm) [inline core/src/main/scala/com/salesforce/op/stages/impl/tuning/DataCutter.scala]: Please use upper case in the first word, e.g. `Number of rows...` (same applies for other log statements.)

File: core/src/main/scala/com/salesforce/op/stages/impl/tuning/OpCrossValidation.scala [hunk 3]
```diff
@@ -136,6 +139,7 @@ private[op] class OpCrossValidation[M <: Model[_], E <: Estimator[_]]
     // Await for all the evaluations to complete
     val modelSummaries = SparkThreadUtils.utils.awaitResult(Future.sequence(modelSummariesFuts.toSeq), maxWait)
 
+    log.info(s"finding best model")
```
Comment (tovbinm) [inline core/src/main/scala/com/salesforce/op/stages/impl/tuning/OpCrossValidation.scala]: is this important?

File: core/src/main/scala/com/salesforce/op/stages/impl/tuning/DataCutter.scala [hunk 4]
```diff
@@ -151,13 +151,15 @@ class DataCutter(uid: String = UID[DataCutter]) extends Splitter(uid = uid) with
   override def validationPrepare(data: Dataset[Row]): Dataset[Row] = {
 
     val dataPrep = super.validationPrepare(data)
+    log.info(s"count of dataset before down-sampling for this fold ${dataPrep.count()}")
```
Comment (tovbinm) [inline core/src/main/scala/com/salesforce/op/stages/impl/tuning/DataCutter.scala]: 1. counts in spark are not free. do you really need it `info`? perhaps in debug only?
2. also use `log.info("count blah blah {}", data.count())` that way if logging is disabled there is no cost for the count

**Comment by gerashegalov** [comment]
I assume we don't need this anymore, feel free to reopen @AdamChit if needed

**Pull request closed without merging**